test/commands/submission/representation/process_test.rb

```diff
@@ -111,6 +111,10 @@ class Submission::Representation::ProcessTest < ActiveSupport::TestCase
     job = create_representer_job!(submission, execution_status: 200, ast:)
     cmd = Submission::Representation::Process.new(job)
     cmd.expects(:handle_generated!).raises
+
+    # We have a guard to reraise in dev/test here, so
+    # stimulate production for this step
```

Comment:
stipulate? Simulate? (I'm sure this is obvious if you're in the right frame of mind but I can't for the life of me figure out what this comment line means)

Reply:
"simulate"

We're basically stubbing Rails' env to say "we're in production, not in test mode".


How about?

```suggestion
    # pretend that we're in production for this step
```

Reply:
Yeah, that makes it clear!